README.md

```diff
@@ -28,24 +28,25 @@ definition. All development is done in the following repositories:
 
 ## Releasing
 
-Since the ``ansys-grantami-serverapi-openapi`` package is auto-generated, the release process differs slightly from
-the standard [PyAnsys release procedure](https://dev.docs.pyansys.com/guidelines/dev_practices.html#release-procedures).
+Since the ``ansys-grantami-serverapi-openapi`` package is auto-generated from the Server API documentation, the 
+versioning scheme will follow the Ansys Granta MI versioning scheme: ``YEAR.RELEASE``, e.g. ``2023.2``.
+
+Release candidates are versioned as ``YEAR.RELEASE.PATCH.rc<ITERATION>``.
```

Comment:
I think there shouldn't be a third `.` in that string? So `YEAR.RELEASE.PATCHrc<ITERATION>`? PEP440 seems to imply that alpha, beta, release candidates don't have a preceding `.`, but dev and post releases do.

Also, will the PATCH ever be != 0 in this situation? I'd think it's always .0rc<ITERATION>, followed by .0, followed directly by .1, .2, if required.

Reply:
True and true. I guess we won't make RCs for patch releases.